EESSI-remove-software.sh

```diff
@@ -101,7 +101,7 @@ fi
 pr_diff=$(ls [0-9]*.diff | head -1)
 
 # if this script is run as root, use PR patch file to determine if software needs to be removed first
-if [ $EUID -eq 0 ]; then
+if [ $EUID -ne 0 ]; then
```

Comment:
This inverts the logic, it now does the opposite of the what the comment suggest. Is that correct?

Reply:
That was necessary when I used a different approach to remove a package. Switched back now to using `fakeroot`.